fluent-package/yum/systemd-test/update-to-next-version-with-active-and-disabled.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+set -exu
+
+. $(dirname $0)/commonvar.sh
+
+# Install the current
+package="/host/${distribution}/${DISTRIBUTION_VERSION}/x86_64/Packages/fluent-package-[0-9]*.rpm"
+sudo $DNF install -y $package
+
+# The service should NOT start automatically
+(! systemctl is-active fluentd)
+# The service should be DISabled by default
+(! systemctl is-enabled fluentd)
+
```

Comment:
It seems "active and enabled" but with-active-and-disable.sh

Reply:
It looks correct because this is the default state directly after installation.
`with-active-and-disable.sh` is about the state before upgrade.